recipes/primesieve/all/conanfile.py

```diff
@@ -0,0 +1,103 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import get, copy, rename, rm, rmdir
+from conan.tools.microsoft import is_msvc_static_runtime, is_msvc
+from conan.tools.scm import Version
+import os
+
+
+required_conan_version = ">=1.53.0"
+
+class PackageConan(ConanFile):
+    name = "primesieve"
+    description = "Fast prime number generator"
+    license = "BSD-2-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/kimwalisch/primesieve"
+    topics = ("math", "prime-numbers", "sieve-of-eratosthenes")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_multiarch": [True, False],
+        "with_msvc_crt_static": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_multiarch": True,
+        "with_msvc_crt_static": False,
+    }
+    package_type = "library"
+
+    @property
+    def _min_cppstd(self):
+        return 11
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def validate(self):
+        if self.settings.compiler.cppstd:
+            check_min_cppstd(self, self._min_cppstd)
+        
+        if self.settings.compiler == "gcc" and Version(self.settings.compiler.version) <= "5":
+            raise ConanInvalidConfiguration("GCC<=5 is currently not supported. Contributions with fixes are welcome.")
```

Comment:
```suggestion
```

I used to work with GCC 4.7 with C++11 minimal support. GCC 4.8 has a really good coverage of C++11. Usually, we don't add restriction to compiler version when the project only requires C++11

Reply:
GCC 5 fails to build, not sure why though. @kimwalisch mentioned above that 4.9 is known not to work, for example.